mlir/test/rocmlir-driver/fp8_ops.mlir

```diff
@@ -1,6 +1,7 @@
 // RUN: rocmlir-gen --arch gfx940 --operation gemm -t fp8 -p | rocmlir-driver --kernel-pipeline=gpu,rocdl | FileCheck %s --check-prefix=MFMA
 // RUN: rocmlir-gen --arch gfx940 --operation gemm -mfma=off -t fp8 -p | rocmlir-driver --kernel-pipeline=gpu,rocdl | FileCheck %s --check-prefix=MFMA_OFF
 // RUN: rocmlir-gen --arch gfx1100 --operation gemm -t fp8 -p | rocmlir-driver --kernel-pipeline=gpu,rocdl | FileCheck %s --check-prefix=GFX11
+// RUN: rocmlir-gen --arch gfx1100 --operation gemm -t fp8 -p | rocmlir-driver --kernel-pipeline=gpu,rocdl | FileCheck %s --check-prefix=GFX11_OCP
```

Comment:
Something's wrong with your tests, because this is the exact same line as the `GFX11` prefix but yet seems to expect different results

Reply:
It should obviously be forcing the types on one of them, and I bet it's making the old ones in both cases, for which the new types will match because they're substrings.  Weakness in the tests.  Anyway, two things to fix.